home/javascript/react/components/FigureProteinSummary.js

```diff
@@ -14,19 +14,19 @@ const FigureSummary = ({statistics, fishID}) => {
         <PublicationCitationLink publication={statistics.firstPublication}/> :
         `${statistics.numberOfPublications} publications`;
 
-    const urlPostfix = `?fishZdbID=${fishID}&expZdbID=${statistics.experiment.zdbID}&geneZdbID=${statistics.antibody.zdbID}&imagesOnly=false`;
+    const urlPostfix = `?fishZdbID=${fishID}&expZdbID=${statistics.experiment.zdbID}&geneZdbID=${markerID}&imagesOnly=false`;
 
     const url = (statistics.experiment && statistics.experiment.standard) ?
         '/action/expression/fish-expression-figure-summary-standard' + urlPostfix :
         '/action/expression/fish-expression-figure-summary-experiment' + urlPostfix;
 
-    if (statistics.numberOfFigures < 10) {
+    if (statistics.numberOfFigures < 5) {
         return <CommaSeparatedList>
             {statistics.publications.map(publication => {
                 return <>{
                     publication.figures.map(figure => {
                         return <CommaSeparatedList key={figure.zdbID}>
-                            <><a href={`/${figure.zdbID}`}>{figure.label}</a>{figure.imgless !== false && <CameraIcon/>}</>
+                            <><a href={`/${figure.zdbID}`}>{figure.label}</a>{figure.imgless !== true && <CameraIcon/>}</>
```

Comment:
why is the logic reversed from previous version? was this meant to be "=== true" ?

Reply:
Because it was wrong. When the figure is not imageLess then we want to show the icon. double negative here. We probably want to change 'imgless' to 'hasImage', that would be more direct.